pkg/logs/internal/tailers/journald/tailer.go

```diff
@@ -51,7 +65,8 @@ func NewTailer(source *config.LogSource, outputChan chan *message.Message) *Tail
 }
 
 // Start starts tailing the journal from a given offset.
-func (t *Tailer) Start(cursor string) error {
+func (t *Tailer) Start(cursor string, journal Journal) error {
+	t.journal = journal
```

Comment:
Would it be possible to pass `journal` as an argument to `NewTailer` instead of setting it here?

Reply:
Yep - good call. It looks like `NewTailer` and `Start` are called in the same place.